.circleci/config.yml

```diff
@@ -12,14 +12,8 @@ jobs:
           command: |
               mkdir -p $HOME/.astropy/config/
               printf "unicode_output = True\nmax_width = 500" > $HOME/.astropy/config/astropy.cfg
-      # In addition to testing 32-bit, we also use the 3.6 and 3.7 builds to
+      # In addition to testing 32-bit, we also use the 3.7 builds to
       # test the ability to run the test suite in parallel.
-      - run:
```

Comment:
@astrofrog , is it okay to remove this, or do I have to replace it with 3.8?

Reply:
I would guess it's OK to ditch it.
Unless there have been any cases where the 3.7 parallel failed but the 3.6 parallel or 3.7 non parallel not, which seems rather unprobable.

Reply:
I agree with @bsipocz